guides/doc-Provisioning_Guide/topics/proc_installing_ipv6_environment.adoc

```diff
@@ -0,0 +1,264 @@
+EOF
+----
+
+== Configuring {Project} for UEFI HTTP Boot IPv6 provisioning
+
+To configure {Project} for UEFI HTTP Boot IPv6 provisioning, system checks must be disabled via `-s` and `--disable-system-checks` options, {SmartProxy} main HTTP endpoint must be enabled (the default port is 8000), TFTP and HTTP boot modules must be enabled as well. Discovery and DNS features can be optionally enabled however DHCP management must be either disabled in the installer or not used meaning that all IPv6 Subnets created in {Project} interface must have DHCP {SmartProxy} set to blank.
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+foreman-installer \
+ -v -s --no-colors \
+ --disable-system-checks \
+ --scenario katello \
+ --foreman-initial-admin-password changeme \
+ --enable-foreman-cli-discovery \
+ --enable-foreman-cli-templates \
+ --enable-foreman-plugin-discovery \
+ --enable-foreman-plugin-templates \
+ --foreman-proxy-http true \
+ --foreman-proxy-httpboot true \
+ --foreman-proxy-tftp true \
+ --foreman-proxy-dhcp false \
+ --foreman-proxy-dns true \
+ --foreman-proxy-dns-zone "example.com" \
+ --foreman-proxy-dns-reverse "c.c.0.0.b.b.b.b.a.a.a.a.0.0.d.f.ip6.arpa" \
+ --enable-foreman-proxy-plugin-discovery \
+ --foreman-initial-admin-password changeme \
+ --foreman-proxy-plugin-discovery-install-images true
+ ----
+
+NOTE: The following paragraph can be a module "How to set up HTTP proxy in {Project}" that can be reused across documents.
+
+If native IPv6 connectivity is not avilable, define HTTP proxy in {Project} and set it as the default proxy in settings.
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+hammer http-proxy create --name "Default proxy" --url http://myproxy.example.com:8888 --organizations "Default Organization" --locations "Default Location"
+hammer settings set --name content_default_http_proxy --value "Default proxy"
+----
+
+NOTE: The following can be a module/procedure. We need to drop a note that this is only relevant when Katello is installed if upstream.
+
+Import a manifest and synchronize {RHEL} 8 kickstart repository.
+
+== Creating new IPv6 Subnet
+
+To create new IPv6 Subnet, go to Infrastructure - Subnets and create new entry:
+
+Protocol: IPv6
+Network Address: valid IPv6 address
+Network Prefix: valid prefix
+Primary DNS Server: must be valid IPv6 address of a DNS server
+
+IPAM modes:
+
+EUI-64 - IPv6 address is calculated from host provisioning interface MAC address
+Internal DB - IPv6 address is selected from a sequence of all available addresses from the subnet
+None - IPv6 address must be entered manually
+
+Boot mode can be set both to DHCP or static, keep in mind that external DHCPv6 server is deployed and when this option is set to static a DHCPv6 service must be deployed on the network for the initial HTTP UEFI bootstrap.
+
+NOTE: Primary DNS server must be present becuase some older versions of {RHEL} 8 do not correctly interper DNS server coming from DHCPv6 response.
```

Comment:
```suggestion
NOTE: Primary DNS server must be present because some older versions of {RHEL} 8 do not correctly interpret DNS servers coming from DHCPv6 responses.
```